internal/docstore/firedocstore/fs_test.go

```diff
@@ -52,7 +52,13 @@ func newHarness(ctx context.Context, t *testing.T) (drivertest.Harness, error) {
 }
 
 func (h *harness) MakeCollection(context.Context) (driver.Collection, error) {
-	return newCollection(h.client, projectID, collectionName, keyName, nil)
+	return newCollection(h.client, projectID, collectionName1, drivertest.KeyField, nil)
```

Comment:
Nice we now have usage of the name and nameFunc in our tests!

Reply:
yup!